pkg/receive_route/hashring.go

```diff
@@ -182,18 +182,3 @@ func HashringFromConfigWatcher(ctx context.Context, updates chan<- Hashring, cw
 		}
 	}
 }
-
```

Comment:
This is a regression I believe. This functionality added by a recent PR https://github.com/thanos-io/thanos/pull/3121
@yashrsharma44

Reply:
Have updated the changes, thanks for identifying this :+1: